src/components/victory-axis.jsx

```diff
@@ -47,9 +47,17 @@ const styles = {
   }
 };
 
-class VAxis extends React.Component {
+@Radium
+export default class VictoryAxis extends React.Component {
+  static role = "axis";
```

Comment:
Well if `child.role` doesn't work than that means the children aren't actually instances of `type` so `child instanceof VictoryAxis` won't work either. What type are the child objects??!